Dockerfile

```diff
@@ -21,4 +20,4 @@ RUN pip install -U pip && \
 COPY . /app
 
 EXPOSE 5000
-CMD ["gunicorn", "--bind=0.0.0.0:5000", "orca:create_app()"]
+CMD ["gunicorn", "--timeout", "600", "--bind=0.0.0.0:5000", "orca:create_app()"]
```

Comment:
Should we have a timeout? What about very large files?

Reply:
I was experiencing some timeout problems with the larger files. I just wanted to add a cap for now that we can revisit later.